text/0000-bdff.md

```diff
@@ -0,0 +1,96 @@
+How will we get there? While many would say that the solution would be to think
+about how other groups run their projects and then synthesizing the best ideas
+this RFC proposes a radical solution to the problem. One that's already been
+with us since the early days of Rust and will continue on beyond our lifetimes.
+
+# Motivation
+[motivation]: #motivation
+
+The motivation behind this RFC is to provide stability to our governance process
+so that we can have a community that it is at ease and feel secure in the
+knowledge that they are in good hands.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+The solution here is to place bors as our BDFF (Benevolent Dictator For Forever)
+as a way to bring a semblance of stability to us all. Consider why:
+
+1. bors has more PRs than any single contributor to rust. They know how to get
+   work done.
+2. They can't be emotionally drained by an endless onslaught of problems being
+   brought up to them. They are code. There is no emotion, therefore they can
+   govern without burn out.
+3. While other BDFLs (Benevolent Dicatator For Life) like the leaders of Linux,
+   vim, and until recently Python, could lead a project for their lifetime, they
+   will eventually pass on. Humans are only organic beings. bors however, could
+   be a BDFF. This would allow them to really consider the longterm view and
+   growth of Rust as a language so that it might exist not only for our
+   descendants, but theirs in perpetuity.
+4. bors works tirelessly for the project 24 hours a day, 7 days a week. They
+   clearly care for the project in ways none of us can possibly hope to match.
+5. We already let bors have stewardship over our repositories and let bors be
+   the one to accept patches into the compiler at the end of the day. With this
+   we would just be formalizing what we already allow in our current project.
+6. bors is an impartial mediator. They have no connections or friends to blind
+   their decisions, they only need make a choice and we can be certain it is
+   correct.
+
+While this might be a radical departure from how other projects are run, we in
+the Rust community have always wanted zero-cost abstractions and what better way
+to represent this in our greatest creation, bors?
+
+A community free of strife and conflict, a community with a singular vision,
+a true arbiter of accepted and rejected PRs, as well as giving our current
+leadership the break they deserve on a more permanent basis. All this and more
+is within our reach by accepting bors as our leader, now and forever.
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+We set `BDFF` to `true` in their code and after that take all guidance from them.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+At most bors turns into a paper clipper maximizer in that they try to
+rewrite the fundamental laws of the universe in Rust in order to optimize
```

Comment:
Niko and @sgrif have been trying to do this for a while now: https://github.com/rust-lang/rust/issues/48536